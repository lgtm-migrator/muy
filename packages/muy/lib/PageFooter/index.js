import React from "react"
import Footer from "../Footer"

const PageFooter = (props) => (
  <Footer
    itemScope
    itemType="https://schema.org/WPFooter"
    role="contentinfo"
    {...props}
  />
)

export default PageFooter
