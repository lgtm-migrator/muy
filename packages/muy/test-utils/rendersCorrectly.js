import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
