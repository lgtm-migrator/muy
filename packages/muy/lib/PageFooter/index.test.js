import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import PageFooter from "."

describe("<PageFooter /> component", () => {
  it("should render correctly", () => {
    rendersCorrectly(<PageFooter />)
  })
})
