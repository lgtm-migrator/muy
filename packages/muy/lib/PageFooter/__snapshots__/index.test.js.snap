// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageFooter /> component should render correctly 1`] = `
<footer
  className="MuiBox-root MuiBox-root-2"
  itemScope={true}
  itemType="https://schema.org/WPFooter"
  role="contentinfo"
/>
`;
